cspell.json

```diff
@@ -119,6 +119,7 @@
     "loglevel",
     "namor",
     "nodataemptystate",
+    "notificationsemptystate",
```

Comment:
Hey,
We don't need to add here as all .avt files have already excluded from spell check.
` "ignorePaths": [
...,
    "*.avt.e2e.js",` ...]